src/darker/import_sorting.py

```diff
@@ -15,11 +15,21 @@
 
 try:
     import isort
+
+    # Work around Mypy problem
+    # https://github.com/python/mypy/issues/7030#issuecomment-504128883
+    isort_code = getattr(isort, "code")
```

Comment:
getattr throws an `AttributeError` when attribute is not available. You probably want to either catch that as well or add a default argument to the `getattr` call

Reply:
![image](https://user-images.githubusercontent.com/17025459/117566350-e78d3d00-b0be-11eb-91fb-92b31393b21b.png)

Reply:
And I just realized you were fixing a different problem